tests/cli-test.js

```diff
@@ -364,4 +364,24 @@ describe('CLI', function () {
       });
     });
   });
+  describe('--config-file', function () {
```

Comment:
can you add a test asserting that we throw if there are conflicts: https://github.com/asakusuma/supported/blob/asa/config-proposal/configuration.md#rules

Reply:
actually should be at least two tests, one for conflicts between primary and custom, and one for conflicts between 2 different custom policies

Reply:
is this a valid config?
```
{
  "custom": [
    {
      "dependencies": ["es6-promise", "@stefanpenner/a", "rsvp"],
      "upgradeBudget": {
        "major": 24,
        "minor": 4,
        "patch": 4
      }
    },
    {
      "dependencies": ["es6-promise"],
      "effectiveReleaseDate": "Dec 10 2022"
    }
  ]
}
```
Here es6-promise repeats but not with different config

Reply:
@SparshithNR no this is not a valid config

> The same package cannot be in multiple CustomPolicy configs

https://github.com/asakusuma/supported/blob/asa/config-proposal/configuration.md#rules

Reply:
> actually should be at least two tests, one for conflicts between primary and custom, and one for conflicts between 2 different custom policies

I will add a test for `conflicts between 2 different custom policies`. Will handle conflict between primary and custom later in different PR as we have to implement the feature to access ignoreDependencies from config file. ignoreDependencies feature is now being implemented in #37 we need to integrate these two changes.